quadratichq/quadratic
# fix: layout bugs on mobile

Currently:

- Command palette shows on mobile
- Permission overlay buttons wrap

We don't want either of these.

![Screenshot 2024-03-28 at 3 55 14 PM](https://github.com/quadratichq/quadratic/assets/1316441/82729f6b-6528-4422-b9ed-183654f1b70d)

To test:

Open [a shared file on the preview branch on mobile](https://quadratic-git-fix-mobile-layout-bugs.vercel.quadratic-preview.com/file/f32db818-2fe5-4861-923d-d9871466a3ca) and see that the command palette icon is gone and the buttons don't wrap.
